src/Symfony/Component/Console/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ CHANGELOG
 7.3
 ---
 
+ * Add `TreeHelper` and `TreeBuilder` to display tree-like structures
+ * Add `TreeStyle` to customize the output of the tree helper
+ * Add `createTree()` and `createTreeBuilder()` methods to `SymfonyStyle`
```

Comment:
I didn't find the `createTreeBuilder` method, maybe to update as you said it is to have feedback :)